libkbfs/keybase_daemon_local.go

```diff
@@ -256,7 +256,17 @@ func (k *KeybaseDaemonLocal) Resolve(ctx context.Context, assertion string) (
 // Identify implements KeybaseDaemon for KeybaseDaemonLocal.
 func (k *KeybaseDaemonLocal) Identify(
 	ctx context.Context, assertion, _ string) (
-	libkb.NormalizedUsername, keybase1.UserOrTeamID, error) {
+	name libkb.NormalizedUsername, utid keybase1.UserOrTeamID, err error) {
+	defer func() {
+		var asUser keybase1.UID
+		asUser, err = utid.AsUser()
+		if err != nil {
+			return
+		}
+		// This is required for every identify call. The userBreak function
+		// will take care of checking if third arg (breaks) is nil or not.
+		getExtendedIdentify(ctx).userBreak(name, asUser, nil)
```

Comment:
@strib @jzila This was needed to make tests not hang. `userBreak` has to be called on every identify. We were already doing that in the real thing, but not in `KeybaseDaemonLocal` yet, which is used in test. Please let me know if you guys see a problem here.

Reply:
(It's only a problem when there's an `extendedIdentify` object attached to `ctx`, which is why it didn't cause any problems for tests before this PR.)